web/src/app/views/about-view/about-view.component.html

```diff
@@ -52,6 +52,44 @@ <h3 class="card-title">Schůzky každý týden</h3>
   </div>
 </section>
 
+<!-- About us -->
+
+<section #bosan class="d-print-none">
+    <div class="container">

```

Comment:
Nainstaluj si do VSCode rozšíření Prettier (https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode), automaticky formátuje odsazení, ať to máme jednotné.

Reply:
Nevím proč, ale už ho tam mám (možná se tam stáhnul při vytvoření toho codespacu)

Reply:
jo to asi jo, ale tak je divný, že to neformátuje 🤷‍♀️